server/trace-func-go/aws/trace_func.go

```diff
@@ -57,7 +57,7 @@ func Handler(_ context.Context, event events.LambdaFunctionURLRequest) (Response
 		return Response{StatusCode: 400}, err
 	}
 
-	standard.IterationsMultiplier = 102 // Cloudlab xl170 benchmark @ 1 second function execution time
+	standard.IterationsMultiplier = 155 // Cloudlab xl170 benchmark @ 1 second function execution time
```

Comment:
AWS has a different iteration multiplier. I don't remember exactly but I think it was around 80